packages/terra-i18n/src/intl.js

```diff
@@ -0,0 +1,4 @@
+import IntlPolyfill from 'intl';
```

Comment:
Does this dependency need to be added to the package.json? I am curious, what does this Polyfill provide? Is this something we should be adding to webpack instead of a global? https://www.npmjs.com/package/intl

Reply:
This dependency is [currently listed in the package.json](https://github.com/cerner/terra-core/blob/master/packages/terra-i18n/package.json#L39).

From the Intl.js readme:
> In December 2012, ECMA International published the first edition of Standard ECMA-402, better known as the ECMAScript Internationalization API. This specification provides the framework to bring long overdue localization methods to ECMAScript implementations.
All modern browsers, have implemented this API. Intl.js fills the void of availability for this API in legacy browsers. It will provide the framework as described by the specification, so that developers can take advantage of the native API in environments that support it, or Intl.js for legacy or unsupported environments.

My preference would be to leave the intl dependency in terra-i18n as-is.